index.html

```diff
@@ -4,7 +4,8 @@
 ---
 
 <h1 class="page-heading">EIPs
-  <a href="https://discord.io/EthCatHerders"><img src="https://dcbadge.vercel.app/api/server/Nz6rtfJ8Cu?style=flat" alt="Discord"></a>
+  <a href="https://discord.io/EthCatHerders"><img src="https://dcbadge.vercel.app/api/server/Nz6rtfJ8Cu?style=flat" alt="Discord channel for Eth R&D eip-editing"></a>
```

Comment:
Isn't this the link for the ECH discord (not "Eth R&D")?

Reply:
Good catch, updated. Thank you @SamWilsn !